### PR TITLE
fix unit test

### PR DESCRIPTION
**Pull request by UDLD**

### CODE REVIEW

File: tests/build.gradle [hunk 1]
```diff
@@ -1,6 +1,9 @@
 description 'Several examples to use cita-sdk-java to interact with CITA'
 
 dependencies {
+
+    testCompile group: 'com.alibaba', name: 'fastjson', version: '1.2.60'
```
Comment (rainchen) [inline tests/build.gradle:5]: is fastjson a new dependent package? is it necessary?

File: .travis.yml [hunk 2]
```diff
@@ -17,7 +17,7 @@ cache:
     - $HOME/.gradle/wrapper/
 
 # Add --info to view details of test failures
-script: ./gradlew --info check jacocoRootTestReport
+script: bash ./initCitaAndInitConfig.sh
```
Comment (rainchen) [inline .travis.yml]: should config as

```
script: 
  - bash ./initCitaAndInitConfig.sh
  - ./gradlew --info check jacocoRootTestReport
```

Comment (rainchen) [inline .travis.yml]: should config as:

```
script:
  # start a CITA chain for running tests
  - bash ./initCitaAndInitConfig.sh
  # run unit and integration tests
  - ./gradlew --info check jacocoRootTestReport
```

Comment (UDLD) [reply .travis.yml]: fixed

**Pull request merged**